src/components/Modal/content/Send.tsx

```diff
@@ -19,18 +28,36 @@ const Send = () => {
   const [recipient, setRecipient] = useState(null);
   const [amountToSend, setAmountToSend] = useState(null);
   const chosenAsset = useSelector(selectChosenAsset);
+  const myaddress = useSelector(selectAccountAddress);
+  const currentTxId = useSelector(selectCurrentTxId);
+  const currentTxErorr = useSelector(selectCurrentTxError);
+  const currentTxStatus = useSelector(selectCurrentTxStatus);
```

Comment:
These values are getting pulled out of the redux only to get passed down as props into the <TxConfirmation /> component, but otherwise they're not being used in <Send /> – we should just pull them out in <TxConfirmation /> directly.

Reply:
I think it can stay like that, TxConfirmation in this case is a component not dependent on the state. We can reuse in the future without modifying it.